docs/ldml/tr35-personNames.md

```diff
@@ -41,26 +41,924 @@ The LDML specification is divided into the following parts:
+
+The output is:
+
+    F. Baz
+
+The “given2” field is empty, so it and the surrounding parentheses are omitted from the output, as is one of the surrounding spaces, according to rule 3. The period after “{given-initial}” remains, because it is separated from the “{given2}” element by  space-- punctuation around a missing field is only deleted up until the closest space in each direction.
+
+If there were no space between the period and the parentheses, as might happen if our hypothetical language didn’t use spaces:
+
+```xml
+<namePattern>{given-initial}.({given2}) {surname}</namePattern>
+```
+
+The input PersonName object still contains:
+
+| `given` | `given2` | `surname` |
+| ------- | -------- | --------- |
+| Foo     |          | Baz       |
+
+The output is:
+
+    F Baz
+
+Both the period after “{given-initial}” _and_ the parentheses around “{given2}” are omitted from the output, because there was no space between them-- instead, we delete punctuation all the way up to the neighboring field. To solve this (making sure the “{given-initial}” field always has a period after it), you would add another namePattern:
+
+```xml
+<namePattern>{given-initial}.({given2}) {surname}</namePattern>
+<namePattern alt=”2”>{given-initial}. {surname}</namePattern>
+```
+
+The first pattern would be used when the “given2” field is populated, and the second pattern would be used when the “given2” field is empty.
+
+Rules 1 and 3 can conflict in similar ways. If the personName element contains (there’s a space between the period and the opening parenthesis again):
+
+```xml
+<namePattern>{given-initial}. ({given2}) {surname}</namePattern>
+```
+
+And the input PersonName object contains:
+
+| `given` | `given2` | `surname` |
+| ------- | -------- | --------- |
+|         | Bar      | Baz       |
+
+The output is:
+
+    Bar) Baz
+
+Because the “given” field is empty, rule 1 not only has us delete it, but also all punctuation up to “{given2}”. This includes _both_ the period _and_ the opening parenthesis. Again, to solve this, you’d supply two namePatterns:
+
+```xml
+<namePattern>{given-initial}. ({given2}) {surname}</namePattern>
+<namePattern alt=”2”> ({given2}) {surname}</namePattern>
+```
+
+The output would then be:
+
+    (Bar) Baz
+
+The first namePattern would be used if the “given” field was populated, and the second would be used if it was empty.
+
+If, instead, the input PersonName object contains:
+
+| `given` | `given2` | `surname` |
+| ------- | -------- | --------- |
+| Foo     |          | Baz       |
+
+The output is:
+
+    F. Baz
+
+### 6.7 <a name="6-7-deriving-initials" href="#6-7-deriving-initials">Deriving initials</a>
+
+The following process is used to produce initials when they are not supplied by the PersonName object. Assuming the input example is “Mary Beth”:
+
+| Action              | Result |
+| ------------------- | ------ |
+| 1. Split into words | “Mary” and “Beth” |
+| 2. Each converted to initials | “M” and “B” |
```

Comment:
Do we need to spell out what "converted to initials" means?  It seems like it might be better to say "Take the first grapheme of each word."

Reply:
Yes, better to say 
Fetch the first grapheme cluster of each word.